app/assets/javascripts/modules/remove-filter.js

```diff
@@ -46,7 +46,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     function getInput(removeFilterName, removeFilterValue, removeFilterFacet) {
       var selector = (!!removeFilterName) ? " input[name='" + removeFilterName + "']" : " [value='" + removeFilterValue + "']";
 
-      return $('#' + removeFilterFacet).find(selector);
+      return $(selector);
```

Comment:
Can you explain this change?

Reply:
It just works! 😛 

I went back to try remember what the issue was but discovered an existing issue with "keywords". I remembered that there seemed to be an issue with removing keywords using the facet labels and the reason was that it could not find the div with id "keywords".

I've fixed undone this change and fixed the bigger issue of keywords div wrapping the whole left sidebar. Keywords div now only wraps the search field. See my last commit